package.json

```diff
@@ -78,15 +76,14 @@
     "pre-commit": "^1.2.2",
     "probe.gl": "^0.0.2",
     "raw-loader": "^0.5.1",
-    "react": "^15.4.0",
-    "react-dom": "^15.4.0",
-    "reify": "^0.4.4",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "reify": "0.4.4",
```

Comment:
Does reify need to be pinned now?

Reply:
Trying to update to the latest breaks the build, so yes. I preferred being explicit by pinning it.